src/main/ui/overworld-ui.gd

```diff
@@ -1,23 +1,31 @@
+class_name OverworldUi
 extends Control
 """
 Overlays 2D UI elements over the 3D world.
 
 This includes chats, buttons and debug messages.
 """
 
+signal chat_started
+
+signal chat_ended
+
+var turbo: Turbo
+
+# Characters we're currently chatting with. We try to keep them all in frame and facing Turbo.
+var chatters := []
+
 var _show_fps := false setget set_show_fps, is_show_fps
 var _show_version := true setget set_show_version, is_show_version
-var _chatting := false
 var _chat_library := ChatLibrary.new()
-
 var _interact_pressed := false
```

Comment:
Remove this unused variable

Reply:
Removed. Removed all other `_pressed` variables throughout the application as well, `event.is_echo()` meets this need.